dashboard/src/components/chapter/AddMember.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <Dialog
+    :options="{
+      title: 'Add New Member',
+    }"
+  >
+    <template #body-content>
+      <div class="flex flex-col gap-2">
+        <div class="text-p-base text-gray-700">
+          Enter the username of the new member you want to add to the team.
```

Comment:
Personal thought: I am not in favour of a considerable user list being exposed here. 
If we could do without the dropdown - would be nice. 

The list could potential also have folks who have anonymised their profiles ?
Thoughts on this @rahulporuri ?

![Screenshot 2024-08-10 at 12 36 38 AM](https://github.com/user-attachments/assets/d201bf4a-73b7-42b2-aa21-08410cfb5011)

Reply:
People with anonymous profile still need to be added to the organizer list, right? Should they choose to organize things. 

Moreover, I don't think this is a bad thing, given that there was a plan to create a central page of all Foss profiles. 

Also, is a profile is "Private" people cannot view it even if they visit the profile route.

Reply:
> People with anonymous profile still need to be added to the organizer list, right? Should they choose to organize things.

They need to be - but I just couldn't relate with this visible listing practice on a dashboard. 

Could go with majority opinion though.

Reply:
We needn't make this change in this PR now too - but keep it as a point of discussion, as I would like to hear more thoughts on this. 

Could go with majority opinion on this also.